packages/widget_toolkit_pin/lib/src/lib_pin_code_with_biometrics/blocs/pin_code_bloc.dart

```diff
@@ -135,23 +137,17 @@ class PinCodeBloc extends $PinCodeBloc {
         pinCode != null;
   }
 
-  /// Adds a digit to the pin code, returning the new length of the pin code
-  Future<int> _addDigit(String digit) async {
-    final pinLength = await pinCodeService.getPinLength();
-    if (_pinCode.value.length < pinLength) {
-      _pinCode.add(_pinCode.value + digit);
-    }
-    return _pinCode.value.length;
-  }
-
   // Checks the validity of the pin code
   Future<dynamic> _checkPin(String pinCode, int digits) async {
     final storedPinLength = await pinCodeService.getPinLength();
     if (storedPinLength != 0 && digits == storedPinLength) {
       try {
         final encryptedPin = await pinCodeService.encryptPinCode(pinCode);
         await pinCodeService.savePinCodeInSecureStorage(encryptedPin);
-        return await pinCodeService.verifyPinCode(encryptedPin);
+        final verificationResult =
+            await pinCodeService.verifyPinCode(encryptedPin);
+        _pinCode.value = '';
```

Comment:
Avoid assigning behaviour values in private functions. This should be explicitly implemented

Reply:
Removed